Library/Homebrew/language/python.rb

```diff
@@ -153,7 +153,8 @@ def needs_python?(python)
       def virtualenv_install_with_resources(options = {})
         python = options[:using]
         if python.nil?
-          wanted = %w[python python@2 python2 python3 python@3 pypy pypy3].select { |py| needs_python?(py) }
+          pythons = %w[python python2 python3 pypy pypy3] + Formula.map(&:name).select { |f| f.start_with? "python@" }
```

Comment:
`Formula.map(&:name)` is a nice idea but it takes ~3.4s on my machine. that might be a bit too slow? Can just leave it hardcoded at 3.8 for now; thanks for trying ❤️

Reply:
Ok, no problem, reverted it back